.github/workflows/aomp-shell.yml

```diff
@@ -0,0 +1,21 @@
+name: AOMP Lint
+
+on:
+  push:
+  pull_request:
+  
+jobs:
+  aomp-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up shellcheck
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y shellcheck
```

Comment:
I don't think we want to be running apt-get update & a possible install on every checkout of aomp.   I would drop out this part.  We can just install shellcheck as needed (I've added it to git/apps/openmp/sysadmin/install-3-other for new system setup/refresh)

Reply:
shellcheck is preinstalled.
https://github.com/actions/runner-images/blob/ubuntu24/20250511.1/images/ubuntu/Ubuntu2404-Readme.md#installed-apt-packages